portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase

```diff
@@ -16,15 +16,15 @@ definition {
 		if ("${testPortalInstance}" == "true") {
 			PortalInstances.tearDownCP();
 		}
+		else {
+			Navigator.openURL();
 
-		Page.changeSinglePageTheme(
-			pageName = "Home",
-			tabName = "Look and Feel",
-			themeName = "Classic");
+			Page.changeSinglePageTheme(themeName = "Classic");
 
-		Navigator.gotoBack();
+			Navigator.gotoBack();
 
-		Button.clickPublish();
+			Button.clickPublish();
```

Comment:
We'd probably want to remove the test-theme.war too. Maybe something like:

```
		AntCommand(
			locator1 = "build-test.xml",
			value1 = "delete-portal-theme -Dtheme-name=test-theme");
```
This may also get rid of the need of changing the page theme back to Classic since the page would revert to default theme.